app/enterprise/0.32-x/getting-started/quickstart.md

```diff
@@ -78,10 +78,10 @@ Now that you have Kong EE running you can interact with the Admin API and GUI.
 
 To begin, go to [Adding your API &rsaquo;][adding-your-api]
 
-[CLI]: /latest/cli
-[API]: /latest/admin-api
+[CLI]: /0.13.x/cli
+[API]: /0.13.x/admin-api
 [GUI]: /enterprise/{{page.kong_version}}/admin-gui/
-[datastore-section]: /latest/configuration/#datastore-section
+[datastore-section]: /0.13.x/configuration/#datastore-section
```

Comment:
This section doesn't appear on the preview.

Reply:
https://docs.konghq.com/0.13.x/configuration/#datastore-section